### PR TITLE
Feature/slop 18/submitted slops

### PR DESCRIPTION
**Pull request by jduncan017**
## Describe your changes
This should be ready to go. Just remember that it needs to be merged at the same time as the api updates on the back end!

## Issue ticket number and link
slop-48
https://apiary-edtonomy.atlassian.net/browse/SLOP-48?atlOrigin=eyJpIjoiNTRmZDI1Y2E5MDZmNGRmNWE5MDFjZDQ2M2VjNzU1OWYiLCJwIjoiaiJ9

## Checklist before requesting a review

- [x ] I have performed a self-review of my code
- [ ] If there are changes to components, I have added / updated automated tests
- [ ] I have made any necessary updates to Storybook to accompany these changes
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ x] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings

### CODE REVIEW

File: src/routes/fest-route/fest-sidebar.jsx [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
-import { useCurrentUser, useModals } from "../../hooks";
+import { useModals } from "../../hooks";
+import { CurrentUserContext } from "../../store";
 
 export const FestSidebar = ({ festQuery }) => {
-  const { openModal, closeModal, registerModal } = useModals();
-  const { currentUser } = useCurrentUser();
-
-  const location = useLocation();
+  const { currentUser } = useContext(CurrentUserContext);
```
Comment (skwisgaarr8-9) [inline src/routes/fest-route/fest-sidebar.jsx:10]: do not get `currentUser` information this way. use our hook `useCurrentUser()`

File: src/routes/submit-route/submitted-slops/submitted-slops-route.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import { useContext } from "react";
+import { Link } from "react-router-dom";
+import { Footer, Header } from "../../../components";
+import { CurrentUserContext } from "../../../store/current-user-context";
+import { SubmittedList } from "./submitted-list";
+
+export const SubmittedSlopsRoute = () => {
+  const { currentUser } = useContext(CurrentUserContext);
```
Comment (skwisgaarr8-9) [inline src/routes/submit-route/submitted-slops/submitted-slops-route.jsx:8]: do not get `currentUser` information this way. use our hook `useCurrentUser()`

**Review by skwisgaarr8-9 — CHANGES_REQUESTED** [review]

**Comment by skwisgaarr8-9** [comment]
you also need to remove `export * from "./SubmitList";` from `src/components/index.js` since that file does not exist anymore

File: src/components/index.js
Comment (mnunezsa95) [inline src/components/index.js:1]: This is throwing an error on my side so I commented it out and it works fine. Not sure if it is needed or if the relative path is just wrong

Comment (jduncan017) [reply src/components/index.js:1]: Hey I'm not sure what you're referring to here

**Comment by jduncan017** [comment]
Main merged and app tested. All requested changes made. Let me know if there's anything else!

**Comment by mnunezsa95** [comment]
> you also need to remove `export * from "./SubmitList";` from `src/components/index.js` since that file does not exist anymore

Resolved

**Review by SamPage4427 — APPROVED** [review]
Everything looks good. Submitted slops show up and the user, if they are an Admin, can approve or deny any of the slop requests. The creator can also update/delete their movie. Slop preview opens when a movie is clicked. Publish and delete send API requests to the backend to change status to either published or delete the movie from the backend. 
Backend is also updated to handle the change in movie status.
Ticket is complete

**Review by skwisgaarr8-9 — APPROVED** [review]

**Pull request merged**